.github/workflows/gh-ci-cron.yaml

```diff
@@ -253,22 +261,33 @@ jobs:
   pypi-latest-release:
     # A set of runner to check that the latest conda release works as expected
     if: "github.repository == 'MDAnalysis/mdanalysis'"
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}-latest
     timeout-minutes: 60
     strategy:
         fail-fast: false
         matrix:
-          os: [ubuntu-latest, macos-latest, windows-latest]
+          os: [ubuntu, macos, windows]
           python-version: ["3.9", "3.10", "3.11"]
+          wheels: ['true', 'false']
```

Comment:
For review context - we want to try to see what happens if you build either from the tarball or wheels, hence the need for this matrix entry.